### ISSUE TITLE
Demons of Asteborg still doesn't work on latest FW

### ISSUE THREAD

**Issue by L10N37** [analoguejb/Analogue-Mega-Sg-JB#1]
ROM is still not working on latest JB FW

**Comment by scottlinux** [comment]
I wanted to give more info that I can confirm I am able to play both a physical retail cart of Demons of Asteborg, and a rom via a Mega Everdrive Pro on this latest jb firmware v7.9 without issue.

However, attempting to play the rom via sd card with the jb genesis core results in a loop at the first level screen:

![asteborg](https://github.com/analoguejb/Analogue-Mega-Sg-JB/assets/2176591/4e52b981-e2d2-48f4-ae04-5c3215a6940b)

**Comment by L10N37** [comment]
Yeah i had this originally, and he just keeps falling in the bad graphics.
With latest firmware i had a white screen lock up.

They do seem to work from mega everdrive pro. A hack was released involving just changing a couple bits or bytes in the header with a hex editor. This was the fix for the mega everdrive anyway.

**Comment by vanfanel** [comment]
I can confirm this same issue with latest JB 7.9 FW.

**Comment by bferrara** [comment]
Tested with Cartridge version of Demons of Asteborg

If I boot the system with 7.9 and

1. Select Run Cartridge
2. Neofid Studio Intro Screen
3. Black (nothing further)

If I

1. Select Cores
2. Select Genesis
3. Back out of the SD card file menu all the way to top menu
4. Select Run Cartridge
5. Game works (haven't played past first few gameplay screens)

I also tested Astebros ROM and it loads to an all white or corrupted visual and goes no further.

Edit: After further testing, Demons Cartridge does not work consistently with any method. It doesn’t work more often than it does, but sometimes it will get past first screen and be playable.
